README.md

```diff
@@ -1,2 +1,7 @@
 # Juntos
 Juntos is a web application that lets you co watch streaming service such as youtube. More services to be added in future such as Netflix, Prime, HBO NOW ...etc.
+
+
+## <a href="https://github.com/pesto-students/juntos-ninja-2-beta/projects/1#card-61704804">Task</a>
+
+### <a href="https://docs.google.com/document/d/1v-cClFokECyzDy7hLe7unWzR4k2S0758zhU-uglkBiY/edit">Youtube API: seekTo usage</a>
```

Comment:
Please put this inside the design doc instead

Reply:
Using readme now.

Reply:
Hey I meant the link to this new doc can be put inside the main design doc

The contents of this PR can live inside `system-design/yt-seek-to.md`

Reply:
Sure